### PR TITLE
Wallet: Fix the endpoint /wallet/transactions/pending/:address

### PR DESCRIPTION
**Pull request by MichaelKim20**
I found the remittance amount incorrect.
I fixed it.

### CODE REVIEW

File: src/modules/storage/LedgerStorage.ts [hunk 1]
```diff
@@ -2408,50 +2408,49 @@ export class LedgerStorage extends Storages {
...
-                    ORDER BY O.amount DESC LIMIT 1
-                )
-            END, TX.address) AS address
+            TF.tx_hash,
+            TF.time,
+            TF.type,
+            TF.tx_fee,
+            TF.payload_fee,
+            TF.received_height,
+            TF.current_height,
```
Comment (TrustHenry) [inline src/modules/storage/LedgerStorage.ts]:
```suggestion
      (SELECT IFNULL(MAX(height), 0) AS height FROM blocks) as current_height,
```

File: src/modules/storage/LedgerStorage.ts [hunk 2]
```diff
@@ -2408,50 +2408,49 @@ export class LedgerStorage extends Storages {
...
                 T.tx_hash,
                 T.time,
                 T.type,
-                S.address,
                 T.tx_fee,
                 T.payload_fee,
                 T.received_height,
-                0 as income,
-                IFNULL(SUM(S.amount), 0) AS spend
+                (SELECT IFNULL(MAX(height), 0) AS height FROM blocks) as current_height,
```
Comment (TrustHenry) [inline src/modules/storage/LedgerStorage.ts]:
```suggestion
```

**Comment by MichaelKim20** [comment]
@TrustHenry I fixed it. Thank you.

**Review by TrustHenry — APPROVED** [review]
LGTM

**Pull request merged**